### DISCUSSION TITLE
Accessibility Support

### DISCUSSION THREAD

**Discussion by chandra-gopalasamy** [terrastruct/d2#2029] [Q&A]
Does D2 support accessibility features? If not, is there a plan to support it and any timeline?

**Comment by alixander** [comment]
Hi, can you be more specific on what features you're looking for?

**Comment by chandra-gopalasamy** [comment]
Hi, thanks for the quick response. I am looking for any support for accessibility options like ARIA attributes, description tags, etc., enabling generated diagrams to be accessible to physically challenged users who use assistive technologies.

**Reply by alixander** [reply]
Hmm interesting, I suppose the diagram can include a description of what's connected to what and what is a parent of what. I'll admit the use case is nebulous to me, especially when the diagram gets more than a handful of nodes. It'd be hard for someone to hear that from a screen reader and keep a mental image in their head.

I'll leave this issue open though as I don't know the space well and maybe there's useful support we can provide.
